horace_core/Tobyfit/tobyfit_DGfermi_resconv_init.m

```diff
@@ -258,15 +258,54 @@
         sample_coords_to_spec_to_rlu(wtmp.experiment_info);
     
     % Get detector information
-    % Because detpar only contains minimal information, hardwire in the detector type here
+    % Because detpar only contains minimal information, either hardwire in 
```

Comment:
Slightly worrying repetition, but this is in the original and due to be sorted as part of the whole `tobyfit` refactor.

Reply:
@oerc0122 unclear what the repetition is